docs/source/about/advanced_usage.rst

```diff
@@ -237,6 +273,40 @@ always_send_scancodes
 
       always_send_scancodes = enabled
 
+high_resolution_scrolling
```

Comment:
I wonder if we should think about having app specific settings. This seems like one example where that could be beneficial.

Reply:
We definitely should